### DISCUSSION TITLE
Lost Sound

### DISCUSSION THREAD

**Discussion by jdycus2558** [smplayer-dev/smplayer#765] [Q&A] [answered]
I was playing some MP4 videos yesterday, and all of a sudden, SMPlayer lost sound. Never could get it back. Any ideas?

**Comment by Raza-0007** [comment, marked as answer]
Check SMPlayer log here "View>SMPlayer log" to see if you get an error message: 

If you have not made a lot of modifications to SMPlayer settings, then the quickest way to recover from an unknown error is to uninstall SMPlayer and reinstall it.
